samplesheets/views.py

```diff
@@ -2632,6 +2863,34 @@ def get_queryset(self):
         # For regular users, dispaly their own requests regardless of status
         return queryset.filter(user=self.request.user)
 
+    def build_webdav_url(self, item):
+        return f"{settings.IRODS_WEBDAV_URL}/{item.path}"
```

Comment:
Double quotes spotted :) F-strings aren't exactly SODAR convention due to historical reasons, but eventually we should migrate SODAR Core and SODAR to them where applicable, so that part is fine!